src/main.cpp

```diff
@@ -174,80 +162,6 @@ namespace
         return outputs;
     }
 
-    std::unique_ptr<ebi::vcf::Parser> build_parser(std::string const & path, ValidationLevel level, ebi::vcf::Version version)
```

Comment:
Seems pretty sensible to move this into the validator namespace, since all the command line arguments have been already converted into objects of the business model.